OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -190,7 +190,7 @@ public class PodCommsSession {
     ///     - PodCommsError.noResponse
     ///     - PodCommsError.podFault
     ///     - PodCommsError.unexpectedResponse
-    ///     - PodCommsError.commandError
+    ///     - PodCommsError.illegalMessage
```

Comment:
```suggestion
    ///     - PodCommsError.invalidMessage
```
I did like rejectedMessage a bit better; invalid sounds like something that fails our idea of invalid and might be an error our comms layer could generate, whereas 'rejected' communicates it failed some idea of validity on the recipient's side.